client/src/components/App.js

```diff
@@ -22,23 +22,23 @@ const App = (props) => {
         setCurrentUser(null)
       })
   }, [])
-  
+
   return (
     <Router>
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-        <div>
-          <div className="bg-img">
-          <div className="bg-container">
-            <div className="bg-text">
-              <h1>HIKEROO</h1>
-              <h2>All the best trails in Massachusetts</h2>
-              <h3>Let's Go Outside!</h3>
-            </div>
+          <div>
+            <div className="bg-img">
+              <div className="bg-container">
+                <div className="bg-text">
+                  <h1>HIKEROO</h1>
+                  <h2>All the best trails in Massachusetts</h2>
+                  <h3>Let's Go Outside!</h3>
```

Comment:
Right now these headers are being displayed on every single page below the topBar and before your content - which I don't think is what you're trying to accomplish here. If you want this content for a home page you should create a home page component to display this text. You would want to render that home page component at the root route `"/"` that you defined on line 30, but it's not rendering a component